Sources/WinAppDriver/WinAppDriver.swift

```diff
@@ -66,6 +122,9 @@ public class WinAppDriver: WebDriver {
                 assertionFailure("WinAppDriver did not terminate within the expected time: \(error).")
             }
         }
+        if let childStdinHandle {
+            CloseHandle(childStdinHandle)
```

Comment:
Do we need to keep the stdin write handle until here if just to close it? And since we have it, we could send `\n` for nicer termination since WinAppDriver awaits one:

```
C:>"C:\Program Files (x86)\Windows Application Driver\WinAppDriver.exe"
Windows Application Driver listening for requests at: http://127.0.0.1:4723/
Press ENTER to exit.

Exiting...

Reply:
If we close the handle, this closes `stdin` for the `WinAppDriver` process, which causes the process to shut down. That's why we need to keep it open until the process shuts down. While we could use that as a proxy to shut down `WinAppDriver` in a cleaner manner, we only have `childStdinHandle` set up if we redirect the output to a file. I'm not opposed to changing the logic but then we have no way of redirecting the output to the spawned terminal to preserve API compatibility.